docs/source/user-guide/productions.md

```diff
@@ -1,31 +1,47 @@
-Simtools is designed to use the CTAO workload management system, but it is flexible enough to work with other job engines such as Grid Engine and HTCondor.
+Simtools is designed to use the CTAO workload management system (WMS) and the HT Condor job submission system.
 
 Pre-requisites for running productions:
 
-- the required simulation software (typically CORSIKA and sim_telarray) are installed on the system (findable through `$SIMTOOLS_SIMTEL_PATH`)
-- a complete simulation model is accessible through the simulation model database
-- production configuration exists (see e.g., [simulate_prod_gamma_20_deg_North.yml](../tests/integration_tests/config/simulate_prod_gamma_20_deg_North.yml))
-- simtools is installed and configured on the job submission host
+- simtools is installed and configured on the job submission host.
+- simulation software (CORSIKA and sim_telarray) are installed on the worker nodes or installed in a container (findable through `$SIMTOOLS_SIMTEL_PATH`) .
+- simulation models are accessible through the simulation model database. This includes both the production tables and the model parameters.
+- configuration parameters for the production (see e.g., [simulate_prod_gamma_20_deg_North.yml](../tests/integration_tests/config/simulate_prod_gamma_20_deg_North.yml))
 
-Productions are configured and submitted using the `simtools-simulate-prod` command.
-The submit engine is configured with the `--submit_engine` command line option.
+Productions are configured and submitted using the
 
-## Running local production
+## Local productions
 
-Choose with `SUBMIT_ENGINE: local` (default value).
+Configure and submit a local production using the `simtools-simulate-prod` command.
+The submit engine is configured with the `--submit_engine local` command line option, see
+[simulate_prod_gamma_20_deg_North.yml](../tests/integration_tests/config/simulate_prod_gamma_20_deg_North.yml) for an example configuration.
 
-Running a local production is a simple way to test the production system, but too slow for anything else.
+Running a local production is a simple way to test the production system, but in general too slow to simulate a large number of events.
 
-## Running productions using HCCondor or Grid Engine
+## Running simtools on HTCondor using Apptainers
 
-Choose with `SUBMIT_ENGINE: htcondor` or `SUBMIT_ENGINE: gridengine`.
+Simtools can be run using HTCondor and Apptainers. Jobs are configured with the `simtools-simulate-prod-htcondor-generator` command.
+Configuration is similar to the `simulate-prod` command, with the following additions:
 
-HTCondor can be configured with additional options, e.g. `SUBMIT_OPTIONS: "priority = 50"`
+- `APPTAINER_IMAGE` is the path to the Apptainer image to be used for the simulation.
+- `PRIORITY` is the priority of the job in HTCondor.
+- `RUN_NUMBER_START` and `NUMBER_OF_RUNS` are the first run number and the number of runs to be simulated.
```

Comment:
Might as well modify this to the new names in #1562, no?

Reply:
I don't understand the change in #1562 for RUN_NUMBER_OFFSET and the HT Condor. There is no need for an offset, here we really give the number of runs to be simulated (and I thought we agreed on keeping this in our call).

Reply:
The offset is essentially always zero for HTCondor, but you could specify it if needed (same use case as DIRAC I guess). The `number_of_runs` stayed as before, it was not removed or changed. Now instead of `run_number_start`, just specify `run_number`.

Reply:
All good - updated the document (and approved #156).